frontend/javascripts/dashboard/dataset/dataset_import_view.js

```diff
@@ -135,7 +136,16 @@ class DatasetImportView extends React.PureComponent<Props, State> {
         this.props.datasetId,
       )) || {
         layers: _.fromPairs(
-          dataSource.dataLayers.map(layer => [layer.name, defaultConfigPerLayer]),
+          dataSource.dataLayers.map(layer => {
+            // Here we adjust the default intensity range if we have a uint16 layer.
+            // This is because otherwise the default max value of the intensity range would be 255.
+            // And this is far to small for uint16.
+            const currentDefaultLayerConfig = _.clone(defaultConfigPerLayer);
+            const elementClass = getElementClass(dataset, layer.name);
+            currentDefaultLayerConfig.intensityRange =
+              elementClass === "uint16" ? [0, 65535] : [0, 255];
```

Comment:
This is equal to https://github.com/scalableminds/webknossos/pull/4381/files#diff-25a4aed2f9763a24b715ea002aed1d63R96. Can you extract this into a function like  `getIntensityRangeForElementClass`?